.gitignore

```diff
@@ -28,7 +28,6 @@ render.experimental.xml
 
 # Keystore files
 *.jks
-*.keystore
```

Comment:
I think this should remain and instead an exception should be added for debug keystore:
```
!debug.keystore
```